Podfile

```diff
@@ -0,0 +1,18 @@
+# Uncomment this line to define a global platform for your project
```

Comment:
Project is using cocoapods now. Cocoapods is managing external libraries, which does not have to populate repository. 
`pod install` needs to be called before project is build.